src/main/java/com/spotinst/sdkjava/model/bl/ocean/ecs/OceanEcsClusterLaunchSpecRequest.java

```diff
@@ -0,0 +1,76 @@
+package com.spotinst.sdkjava.model.bl.ocean.ecs;
```

Comment:
Create a folder 'requests' and move this file under that folder instead of being under bl. Keep the folder structure as .model.requests.ocean.ecs